src/components/header-card-action-buttons/header-card-action-buttons.tsx

```diff
@@ -39,64 +39,72 @@ export const HeaderCardActionButtons: FC<Props> = ({ token }) => {
       ? `You need to have ${metadata.symbol} to pay gas fee`
       : 'Balance is zero';
 
-  const disableSendAsset = token.balance === emptyToken.balance || tezosToken.balance === emptyToken.balance;
-
   const actionButtonStylesOverrides = useMemo(
     () => ({
       titleStyle: styles.actionButtonTitle
     }),
     [styles.actionButtonTitle]
   );
-  const handleTouchStart = () => {
-    if (disableSendAsset) {
-      showErrorToast({ description: errorMessage });
-      dispatch(setOnRampPossibilityAction(true));
+
+  const emptyBalance = token.balance === emptyToken.balance || tezosToken.balance === emptyToken.balance;
+  const disabledSendButton = emptyBalance && isIOS;
+
+  const handleSendButton = () => {
+    if (!emptyBalance) {
```

Comment:
no, because it will be working wrong for Android because we have `isIOS` in that condition.
it means that on Android `!disabledSendButton` will have `true` when `emptyBalance` is `0` because `isIOS` will be `false`.